create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -219,9 +220,15 @@ private UnsupportedCitationTypeException getUnsupportedCitationTypeException() {
 
     private Optional<PublicationInstance<? extends Pages>> convertCitationTypeToPublicationInstance(
         CitationtypeAtt citationtypeAtt) {
-        return CitationtypeAtt.AR.equals(citationtypeAtt)
-                   ? Optional.of(new JournalArticle())
-                   : Optional.empty();
+        switch (citationtypeAtt) {
+            case AR:
+                return Optional.of(new JournalArticle());
+            case BK:
+            case CH:
+                return Optional.of(new BookMonograph());
+            default:
+                return Optional.empty();
+        }
```

Comment:
Actually I am missing a bit clarification what those codes mean. Jeg vet hva de betyr i dag, men vet noen nye folk det?
Hvordn få vi det til? Enten wrapper dem i et eget Enum med fortolking (ikke så lurt kanskje) eller just simple plain comments? Hva tror dere?

Reply:
I think that wrapping the enum in another enum is not the best solution. We are generating the CitationTypeAtt class from the xsd which we fetch on code build. I suggest a reference to the scopus documentation pdf? I we absolutely need to comment.

Reply:
Does the commnet look okay Gregor?